tools/dataset_converters/images2coco.py

```diff
@@ -0,0 +1,100 @@
+import argparse
+import os
+
+import mmcv
+from PIL import Image
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Convert images to coco format without annotations')
+    parser.add_argument('img_path', help='The root path of images')
+    parser.add_argument(
+        'classes', type=str, help='The text file name of storage class list')
+    parser.add_argument(
+        'out',
+        type=str,
+        help='The output annotation json file name, The save dir is in the '
+        'same directory as img_path')
+    parser.add_argument(
+        '-e',
+        '--exclude-extensions',
+        type=str,
+        nargs='+',
+        help='The suffix of images to be excluded, such as "png" and "bmp"')
+    args = parser.parse_args()
+    return args
+
+
+def collect_image_infos(path, exclude_extensions=None):
+    img_infos = []
+
+    images_generator = mmcv.scandir(path, recursive=True)
+    for image_path in mmcv.track_iter_progress(list(images_generator)):
+        if exclude_extensions is None or (
+                exclude_extensions is not None
+                and not image_path.lower().endswith(exclude_extensions)):
+            image_path = os.path.join(path, image_path)
+            img_pillow = Image.open(image_path)
```

Comment:
suggest to use mmcv.imread which is faster

Reply:
`Image.open` just get the handle, not really read the picture. So it should be the fastest.